### PR TITLE
serbian-language-translation

### PR DESCRIPTION
**Pull request by lxmarinkovic**
Hi @mojombo , @JohnTitor  , @alexandrtovmach , @steveklabnik , @indirect , @anangaur  & @segiddins  here is a translation to the serbian language. Pls review and merge when you find time for it :D

### CODE REVIEW

File: lang/sr/spec/v2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+
+O projektu
+----------
+
+Autor specifikacije Semantičkog Verzionisanja je [Tom
+Preston-Werner](http://tom.preston-werner.com), pronalazač Gravatar-a i suosnivač GitHub-a.
+
+Ako želite ostaviti povratne informacije, molimo [otvorite issue na
+GitHub-u](https://github.com/mojombo/semver/issues).
```
Comment (JohnTitor) [inline lang/sr/spec/v2.0.0.md]: Please point to the original repo.

```suggestion
GitHub-u](https://github.com/semver/semver/issues).
```

File: lang/sr/index.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+
+O projektu
+----------
+
+Autor specifikacije Semantičkog Verzionisanja je [Tom
+Preston-Werner](http://tom.preston-werner.com), pronalazač Gravatar-a i suosnivač GitHub-a.
+
+Ako želite ostaviti povratne informacije, molimo [otvorite issue na
+GitHub-u](https://github.com/mojombo/semver/issues).
```
Comment (JohnTitor) [inline lang/sr/index.md]: Ditto

**Comment by JohnTitor** [comment]
The translation looks good with a machine translation tool. Do you have any other Serbian speakers helping review here? If not, that's totally fine, but a review from a native speaker would be helpful.

File: lang/sr/index.md (hunk 2 above)
Comment (lxmarinkovic) [reply lang/sr/index.md]: ✅

**Comment by lxmarinkovic** [comment]
NATM

**Comment by lxmarinkovic** [comment]
> The translation looks good with a machine translation tool. Do you have any other Serbian speakers helping review here? If not, that's totally fine, but a review from a native speaker would be helpful.

NATM

**Comment by lxmarinkovic** [comment]
> The translation looks good with a machine translation tool. Do you have any other Serbian speakers helping review here? If not, that's totally fine, but a review from a native speaker would be helpful.

NATM

**Comment by lxmarinkovic** [comment]
> The translation looks good with a machine translation tool. Do you have any other Serbian speakers helping review here? If not, that's totally fine, but a review from a native speaker would be helpful.

Thank you! 😃 NATM (on Serbian speaking reviewer) 😐

**Comment by lxmarinkovic** [comment]
Hi again, @JohnTitor , I have searched through the contributors list, AFAIK there are no serbian speaking maintainers who can help with the review 🙀

**Comment by lxmarinkovic** [comment]
Hi again, @JohnTitor , I have searched through the contributors list here, AFAIK there are no serbian speaking maintainers who can help with the review 🙀

**Comment by JohnTitor** [comment]
Sorry for the delay! Alright, the machine translation doesn't seem to be weird so I'm going to accept.
Could you fix the lint failure?

**Comment by lxmarinkovic** [comment]
> Sorry for the delay! Alright, the machine translation doesn't seem to be weird so I'm going to accept. Could you fix the lint failure?

Hi @JohnTitor , thank you for getting back! Lint is fixed now 😀

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thank you!

**Pull request merged**